src/main/scala/zio/prelude/Associative.scala

```diff
@@ -89,14 +89,13 @@ object Associative extends Lawful[AssociativeEqual] {
     }
 
   /**
-   * The `Commutative` and `Inverse` instance for the product of `Boolean`
+   * The `Commutative` and `Identity` instance for the product of `Boolean`
    * values.
    */
-  implicit val BooleanProdCommutativeInverse: Commutative[Prod[Boolean]] with Identity[Prod[Boolean]] =
-    new Commutative[Prod[Boolean]] with Inverse[Prod[Boolean]] {
+  implicit val BooleanProdCommutativeIdentity: Commutative[Prod[Boolean]] with Identity[Prod[Boolean]] =
+    new Commutative[Prod[Boolean]] with Identity[Prod[Boolean]] {
       def combine(l: => Prod[Boolean], r: => Prod[Boolean]): Prod[Boolean] = Prod(l && r)
       val identity: Prod[Boolean]                                          = Prod(true)
-      def inverse(l: => Prod[Boolean], r: => Prod[Boolean]): Prod[Boolean] = Prod(l || !r)
```

Comment:
why are we getting rid of this? this doesn't divide by zero? Or did it break the laws?

Reply:
See comment above.